.github/workflows/main.yml

```diff
@@ -73,7 +72,7 @@ jobs:
 
   tag-and-publish:
     name: Build Linux Binaries and Docker Image, Tag, and Publish
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
will this cause a problem with glibc? i think it will? i'd feel better if we kept this at 18

Reply:
actually would you mind reverting this for now? thx. i forgot about this comment and merged it... :)